### PR TITLE
PlatformFaultDomain for VmProfile

### PR DESCRIPTION
**Pull request by grizzlytheodore**
from API owner: @shpa-microsoft

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [x] Removing API(s) in stable version
- [x] Removing properties in stable version
- [x] Removing API version(s) in stable version
- [x] Updating API in stable version with Breaking Change Validation errors
- [x] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by grizzlytheodore** [comment]
added 'DoNotMerge' label to follow new API version PR process for Cplat

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json [hunk 1]
```diff
@@ -8940,6 +8940,11 @@
           "$ref": "#/definitions/SubResource",
           "description": "Specifies information about the proximity placement group that the virtual machine should be assigned to. <br><br>Minimum api-version: 2018-04-01."
         },
+        "platformFaultDomain": {
```
Comment (markcowl) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json:8943]: Can we add this to existing examples?

**Review by markcowl — CHANGES_REQUESTED** [review]

**Comment by markcowl** [comment]
@grizzlytheodore You will need to fix the LintDiff error above (see https://aka.ms/ci-fix) when this is ready to merge

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json (hunk 1 above)
Comment (grizzlytheodore) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json:8943]: @shpa-microsoft

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/runCommands.json [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+          "default": false
+        }
+      },
+      "required": [
+        "name",
+        "type"
+      ],
+      "description": "Describes the properties of a run command parameter."
+    },
+    "RunCommandDocumentBase": {
```
Comment (markcowl) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/runCommands.json:323]: Is this a resource?

**Review by markcowl — CHANGES_REQUESTED** [review]

**Review by pilor — APPROVED** [review]

**Comment by shpa-microsoft** [comment]
Based on private chat; Abandoning this with new PR https://github.com/Azure/azure-rest-api-specs/pull/11409

**Comment by Sandido** [comment]
Closing as per above comment. New PR 11409.

**Pull request closed without merging**